simulator/index.html

```diff
@@ -7,6 +7,7 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"
           integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7"
           crossorigin="anonymous" />
+    <link rel="stylesheet" href="../css/simulator.css" />
```

Comment:
It looks like you're removing file ../css/simulator.css from the code base, why is a reference getting added in the markup?